### PR TITLE
Allow repository maintainers to force-merge

### PR DESCRIPTION
**Pull request by llucax**
The repository ruleset `Protect version branches` has been updated to allow repository maintainers to skip protection rules in PRs.

### CODE REVIEW

**Comment by llucax** [comment]
It seems like this will not work until merge queues are supported by rulesets.

* https://github.com/orgs/community/discussions/61107#discussioncomment-7993004

**Review by daniel-zullo-frequenz — DISMISSED** [review]

**Comment by llucax** [comment]
Rebased, needs a new approval.

**Review by shsms — APPROVED** [review]

**Pull request merged**
